src/BIP39Wallet.cs

```diff
@@ -0,0 +1,121 @@
+using NBitcoin;
+using System.Text;
+using AElf.Types;
+using System.Security.Cryptography;
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+
+namespace BIP39Wallet
+{
+    [SuppressMessage("ReSharper", "UnusedAutoPropertyAccessor.Global")]
+    public class Wallet
+    {
+        public class BlockchainWallet
+        {
+            public string Address { get; private set; }
+            public string PrivateKey { get; private set; }
+            public string Mnemonic { get; private set; }
+            public string PublicKey { get; private set; }
+
+            public BlockchainWallet(string address, string privateKey, string mnemonic, string publicKey)
+            {
+                Address = address;
+                PrivateKey = privateKey;
+                Mnemonic = mnemonic;
+                PublicKey = publicKey;
+            }
+        }
+        public virtual string ConvertMnemonicToSeedHex(Mnemonic mnemonic, string password)
+        {
+            var mnemonicBytes = Encoding.UTF8.GetBytes(mnemonic.ToString().Normalize(NormalizationForm.FormKD));
+            var saltSuffix = string.Empty;
+            if (!string.IsNullOrEmpty(password))
+            {
+                saltSuffix = password;
+            }
+            var salt = $"mnemonic{saltSuffix}";
+            var saltBytes = Encoding.UTF8.GetBytes(salt);
+
+            var rfc2898DerivedBytes = new Rfc2898DeriveBytes(mnemonicBytes, saltBytes, 2048, HashAlgorithmName.SHA512);
+            var key = rfc2898DerivedBytes.GetBytes(64);
+            var hex = BitConverter
+                .ToString(key)
+                .Replace("-", "")
+                .ToLower();
+
+            return hex;
+        }
+
+        public virtual BlockchainWallet CreateWallet(int strength, Language language, string password)
+        {
+            var mnemonic = new Mnemonic(Wordlist.English, WordCount.Twelve);
+
+            var seed = ConvertMnemonicToSeedHex(mnemonic, "");
+            var masterKeyPath = new KeyPath("m/44'/1616'");
```

Comment:
The string can be extracted out and make as constant and give it a proper name.

Reply:
extracted out in BIP39WalletCOnstants.cs, name it as AElfPath